Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue

```diff
@@ -17,25 +17,19 @@
           height="44"
           alt="B.C. Government Logo"
         />
-      </router-link>
+          <b-navbar-brand>
```

Comment:
indentation is off

Reply:
sorry, code noob here! I was relying on an extension to take care of the indentation

Reply:
lintfix should take care of that